examples/private_set_intersection/client/private_set_intersection.cc

```diff
@@ -68,7 +65,13 @@ int main(int argc, char** argv) {
 
   // TODO(#1066): Use a more restrictive Label, based on a bearer token shared between the two
   // clients.
-  oak::label::Label label = oak::PublicUntrustedLabel();
+  // Public key is stored in the `examples/keys/ed25519/test.pub` file.
+  std::string public_key_base64 = "f41SClNtR4i46v2Tuh1fQLbt/ZqRr1lENajCW92jyP4=";
```

Comment:
Could this be read from the file instead of being hard coded here?

Reply:
Done